plan/stats.go

```diff
@@ -47,7 +47,6 @@ func (s *statsInfo) scale(factor float64) *statsInfo {
 	profile := &statsInfo{
 		count:          s.count * factor,
 		cardinality:    make([]float64, len(s.cardinality)),
-		histColl:       s.histColl,
```

Comment:
`scale` is called by `scaleByExpectedCnt` which is called many many times when building physical plan.
It's hard to handle `expectedCnt` with stats estimation by `Histogram`. So I think in such case `HistColl` is not need to maintained. Thus maintain the `HistColl` out of this method.

Reply:
But since it's the problem of `scaleByExpecteCnt`, why not do it in that function?

Reply:
Emm.. i need some clearer comments?